doc/config.example.yaml

```diff
@@ -211,7 +211,7 @@ network:
   - http://192.168.0.44:2828
 dns:
   # Supported value: FQDN seed
-  - seed.bosagora.io
+  - dns://seed.bosagora.io
```

Comment:
I'm not sure that's a QoL improvement. For `network`, it makes sense, since there can be multiple IPs, but for this field, it can only ever be `<domain-or-ip>[:<port>]?`

Reply:
I only care about the `network` config actually, will revert